src/em/prov/em_provisioning.cpp

```diff
@@ -265,14 +339,104 @@ int em_provisioning_t::create_bss_config_req_msg(uint8_t *buff)
+
+    // One Backhaul STA Radio Capabilities TLV.
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_bh_sta_radio_cap;
+    tlv_size = create_bsta_radio_cap_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    // One AP capability TLV
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_ap_cap;
+    tlv_size = create_ap_cap_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    // One AP Radio Basic Capabilities TLV for each of the supported radios of the Multi-AP Agent.
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_ap_radio_basic_cap;
+    tlv_size = create_ap_radio_basic_cap(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    //  One AKM Suite Capabilities TLV
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_akm_suite;
+    tlv_size = create_akm_suite_cap_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    //  One Profile-2 AP Capability TLV.
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_profile_2_ap_cap;
+    tlv_size = create_prof_2_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    //  One BSS Configuration Request TLV with DPP attribute(s) for all supported radios of the Multi-AP Agent.
+    tlv = reinterpret_cast<em_tlv_t *> (tmp);
+    tlv->type = em_tlv_type_bss_conf_req;
+    tlv_size = create_bss_conf_req_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    //  One AP HT Capabilities TLV for each radio that is capable of HT (Wi-Fi 4) operation.
+    tlv = reinterpret_cast<em_tlv_t *>(tmp);
+    tlv->type = em_tlv_type_ht_cap;
+    tlv_size = create_ht_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    //  One AP VHT Capabilities TLV for each radio that is capable of VHT (Wi-Fi 5) operation.
+    tlv = reinterpret_cast<em_tlv_t *>(tmp);
+    tlv->type = em_tlv_type_vht_cap;
+    tlv_size = create_vht_tlv(tlv->value);
+    tlv->len = htons(static_cast<uint16_t>(tlv_size));
+
+    tmp += sizeof(em_tlv_t) + tlv_size;
+    len += sizeof(em_tlv_t) + tlv_size;
+
+    // NOTE: this CMDU is extended in R6 with additional TLVs for Wi-Fi 6/6E and Wi-Fi 7 capabilities.
+
 
     // End of message
     tlv = reinterpret_cast<em_tlv_t *> (tmp);
```

Comment:
Similarly, at the end the EOM code can be replaced by this (I can't add comments beyond the scope of the changes)
```suggestion
    tmp = em_msg_t::add_eom_tlv(tmp, &len);
```